rfc#0001.md

```diff
@@ -0,0 +1,154 @@
+# Wasm binary loading into an Enarx Keep
+- Authors: [Mike Bursell](https://github.com/MikeCamel)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-01-10
+- Status Note: (has been subject to initial discussion and consensus)  
+- Supersedes: n/a
+- Start Date: 2020-02-16
+- Tags: architecture, webassembly
+
+## Summary
+
+This RFC describes the architecture adopted by Enarx for loading Wasm
+packages into a prepared Keep using a secure channel. 
+
+## Motivation
+
+Once a Keep is prepared with shim, Wasm runtime, etc., the
+tenant workload (Wasm binary + data) needs to be loaded into it.  A
+description of the loading sequence is required.
+
+## Tutorial
+
+A Keep is the fundamental runtime component of Enarx: a TEE instance
++ Enarx runtime pieces, including WebAssembly and WASI layers.  In
```

Comment:
This is rendered strangely in markdown. "Enarx runtime pieces" becomes a bullet point.